### PR TITLE
Migrate to @pooltogether/react-components 

### PR DESCRIPTION
**Pull request by ddesrosier**
- Integrate v.0.0.1 of the components library, just using it for tailwind config & css for now.
- Update the app to use `useAppEnv` to get the proper network for users. Now the app handles when a users wallet is on an unsupported network
- Update localization code to match flagship

### CODE REVIEW

File: lib/components/TransactionsList.jsx [hunk 1]
```diff
@@ -23,7 +25,7 @@ export function TransactionsList(props) {
   const handleClearPrevious = (e) => {
     e.preventDefault()
 
-    if ((usersAddress, chainId)) {
```
Comment (chuckbergeron) [inline lib/components/TransactionsList.jsx:26]: that's... weird

**Review by chuckbergeron — CHANGES_REQUESTED** [review]
The Settings menu has "Use testnets" as an option, and that seems to be the switch to change the content from ETH Mainnet to Rinkeby. However I can still be connected to the wrong network and get the other network's content (ie I'm on Mainnet with my wallet and see Rinkeby proposals). We should prob have the same two different arrays as the flagship, and have [Networks.mainnet] in the production one and [Networks.rinkeby] in the testnets one, instead of relying on the 1 `SUPPORTED_CHAIN_IDS` in the constants file.

Also let's make sure you can't create a tx for the wrong network! I see you have `disabled` on the buttons, but in the flagship we also show a tooltip saying you need to change you wallet network. Can prob just copy and paste that code.

The one other thing I noticed is how different the layouts are between the flagship and the vote app now, and how the ETH diamond image doesn't show up in the header but instead it says 'E'.

**Comment by ddesrosier** [comment]
There's a ton of changes now. Majority of components have been migrated, most importantly is the layout, navigation & header. Some specific things to look at:
- layout, navigation & header
- account / pending transactions modal & trigger updates
    - Moved to a single `Modal` component
    - `1 pending` replaces the users address/icon rather than being placed beside
        - I copied Uniswap for this. Mostly so the header isn't shifting around when we send a transaction
- network selection modal
    - https://github.com/pooltogether/pooltogether-react-components/tree/main/lib/components/PageHeader/NetworkSelector 
    - modals are just quicker so I didn't add the dropdown like the designs. I can update that before we release anything
    - There's different content for when Metamask is connected vs another wallet

I've created a proposal , voted & queued on Rinkeby so everything seems to be functioning fine.

Pending tx / account modal
Screen Shot 2021-06-15 at 9 46 54 AM

Page header
Screen Shot 2021-06-15 at 9 46 42 AM
Screen Shot 2021-06-15 at 9 46 01 AM

Network selection modal (with Metamask connected)
Screen Shot 2021-06-15 at 9 46 31 AM
Screen Shot 2021-06-15 at 9 46 23 AM

Network selection modal (with non-metamask connected)
Screen Shot 2021-06-15 at 10 52 44 AM

Settings popout
Screen Shot 2021-06-15 at 9 46 07 AM

**Comment by chuckbergeron** [comment]
The screenshots are looking good! One thing, could we move that tooltip 'i' beside the Ethereum button on the network modal when Ethereum is connected? It doesn't look correct having the buttons two different widths.

One thing we could do is wrap it so it looks like one button.

```
|-----------------------|
| Ethereum              |   i   |
|-----------------------|
```

**Comment by chuckbergeron** [comment]
Getting a few errors in the console:

Screenshot 2021-06-15 at 19 41 01


Screenshot 2021-06-15 at 19 41 11

**Comment by chuckbergeron** [comment]
Could we swap these so they go from this gray color to green? I find links typically go from whatever state they are to something brighter on hover, so this dropping of opacity on hover feels wrong to me:

Screenshot 2021-06-15 at 19 44 24

**Comment by chuckbergeron** [comment]
Same thing for buttons, the hover state looks disabled?

Screenshot 2021-06-15 at 19 44 50

**Comment by chuckbergeron** [comment]
It's in the nav, too:
Screenshot 2021-06-15 at 19 45 20
😂

**Comment by chuckbergeron** [comment]
This could use at least double the padding:
Screenshot 2021-06-15 at 19 47 05


For reference here's what the design looks like in Figma:
Screenshot 2021-06-15 at 19 47 30

**Pull request merged**